### ISSUE TITLE
Stuck on "disabling certificate pinning"

### ISSUE THREAD

**Issue by KJL777** [shroudedcode/apk-mitm#9]
Hey! Appreciate all your hard work making this tool. Just a quick question regarding an issue I am experiencing. I've tried a few different apks and always hang on "disabling certificate pinning". The process never completes. I am wondering whether it is just the particular apks, or something on my end.

Thanks very much.

![issue](https://user-images.githubusercontent.com/52773876/69010402-5d479b00-0957-11ea-95f4-3a21c77e8631.PNG)

**Comment by shroudedcode** [comment]
It seems like you're stuck on [this line of code](https://github.com/shroudedcode/apk-mitm/blob/83716a04b1bd0f75a3254850c7b205dc6e9601f6/src/tasks/disable-certificate-pinning.ts#L22):

```ts
const smaliFiles = await globby(path.join(directoryPath, '**/*.smali'))
```

Looks like globby is taking forever to find all the files ending in `*.smali`.

You could try running globby manually to see it that also takes forever. To do that, open a terminal inside the temporary directory (logged at the top) and run:

```
npx globby-cli 'decode/**/*.smali'
```

**Comment by KJL777** [comment]
Thank you for the reply. Hmm, it must be a careless issue on my end. No luck with the above suggestion. Unsure if what I am doing incorrectly.

![issue2](https://user-images.githubusercontent.com/52773876/69017031-d8cb3b80-099b-11ea-86b8-950aecb3f462.PNG)

**Comment by shroudedcode** [comment]
I just realized that `cmd` doesn't interpret strings in single quotes the same way `bash` does, so please try running the command again with double quotes:

```
npx globby-cli "decode/**/*.smali"
```

**Comment by KJL777** [comment]
![small app](https://user-images.githubusercontent.com/52773876/69096863-366b9080-0a4d-11ea-832b-bebd5a7f86c8.PNG)

I was able to have some success trying globby manually, but this only worked on a small sized apk (calculator) that I tried. Two other large game apks yield the same result was the previous screenshot.

Also, despite globby working manually with the calculator app, it still hangs while attempting locate smali files using your tool.
